bandit/plugins/hashlib_new_insecure_functions.py

```diff
@@ -34,6 +34,10 @@
     CWE information added
 
 """
+import sys
+
+import distutils
```

Comment:
`distutils` is such a nightmare I would rather we implement `strtobool` ourselves instead of relying on it. I believe it is slated to be removed from the stdlib soon too so we will be broken once it does